src/createAnimatedComponent.tsx

```diff
@@ -356,12 +391,13 @@ export default function createAnimatedComponent(
           tag: viewTag!,
           // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
           name: viewName!,
+          shareableNode,
```

Comment:
```suggestion
          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
          shareableNode: shareableNode!,
```
Shouldn't there also be an assert to be consistent?

Reply:
Fixed in 97b5787d5bd3191a22050e0d8a52337e2f3b478f